external/mmdetection/detection_tasks/extension/utils/pipelines.py

```diff
@@ -25,6 +27,12 @@
 from ..datasets import get_annotation_mmdet_format
 
 
+# Clean up cache directory per process launch
+_CACHE_DIR = "/tmp/img-cache"
```

Comment:
tempfile.TemporaryDirectory? Or you intend to save this cache across runs?

Reply:
Can we delete the temporary directory at every training? AFAIK, /tmp directory is cleaned at every booting.
In this PR, since we save lots of images in cache-directory, deleting it at every training is needed. 
And for this case, we can't use tempfile.TemporaryDirectory.

Reply:
Reflected. Please refer #1491